src/main/java/com/splunk/logging/AckLifecycleState.java

```diff
@@ -21,23 +21,49 @@
  */
 public class AckLifecycleState {
```

Comment:
@meemax @anushjay if you want to change this classname to AckLifecycleEvent that would be fine. But maybe we can defer that until we get the merge done, and get rid of this entire logging-library dependency.